### ISSUE TITLE
Dependencies published to Maven2 repo ?? They seem to be missing

### ISSUE THREAD

**Issue by iangregsondev** [bitcoin-sv/block-headers-client#16]
Hi,

I tried to follow the documentation using CORE. (here https://github.com/bitcoin-sv/block-headers-client/tree/master/core)

The JCLVersion is set to 2.0.0 then gradle gives an error that it's not there.

https://repo.maven.apache.org/maven2/io/bitcoinsv/jcl/jcl-store-keyValue/2.0.0/jcl-store-keyValue-2.0.0.pom

But 2.2.0 is i.e.

https://repo.maven.apache.org/maven2/io/bitcoinsv/jcl/jcl-store-keyValue/2.2.0/

Also, this is ALSO missing. 

https://repo.maven.apache.org/maven2/io/bitcoinsv/headerSV/headerSV-core/2.0.0/headerSV-core-2.0.0.pom

And there is no 2.2.2 - in fact the artifact seems to be missing altogether.

https://repo.maven.apache.org/maven2/io/bitcoinsv/headerSV/

Now using

gradle.settings

```
# JCL dependency:
JCLVersion=2.2.0

# headerSV dependency:
headerSVVersion=2.0.0
```

build.gradle ( using kotlin syntax)
```
val kotlinVersion=project.properties.get("kotlinVersion")
val bitcoinJVersion=project.properties.get("bitcoinJVersion")
val JCLVersion=project.properties.get("JCLVersion")
val headerSVVersion=project.properties.get("headerSVVersion")


repositories {
    mavenCentral()
    mavenLocal()
}

dependencies {
    // Removed unimportant things for pasting here......

    // bitcoinJ modules
    implementation("io.bitcoinsv.bitcoinjsv:bitcoinj-base:${bitcoinJVersion}")

    // jcl Modules
    implementation("io.bitcoinsv.jcl:jcl-net:${JCLVersion}")
    implementation("io.bitcoinsv.jcl:jcl-tools:${JCLVersion}")
    implementation("io.bitcoinsv.jcl:jcl-store:${JCLVersion}")
    implementation("io.bitcoinsv.jcl:jcl-store-keyValue:${JCLVersion}")
    implementation("io.bitcoinsv.jcl:jcl-store-levelDB:${JCLVersion}")

    // HeaderSV Modules
    implementation("io.bitcoinsv.headerSV:headerSV-core:${headerSVVersion}")

```

**Comment by iangregsondev** [comment]
I am guessing, maybe I am wrong, but maybe the artifacts are published to the local maven repo and never been published to maven central ?

Any ideas ?

**Comment by Danconnolly** [comment]
This should be fixed but I suspect it will take some time.

**Comment by iangregsondev** [comment]
No worries.

**Comment by Danconnolly** [comment]
This is a documentation error, the correct values are in https://github.com/bitcoin-sv/block-headers-client/blob/main/gradle.properties

**Comment by iangregsondev** [comment]
Yes, the JCL version I found as per my message above by searching inside the repo.

But this URL for the headers client, I only see version 1.

https://repo.maven.apache.org/maven2/io/bitcoinsv/headerSV/block-headers-client/

The information in the gradle file shows the following

    // HeaderSV Modules
    implementation("io.bitcoinsv.headerSV:headerSV-core:${headerSVVersion}")

So the new version info you included a link for will not work as it's physically not there as far as I can see.

**Comment by Danconnolly** [comment]
Oh I see, sorry, missed that.

**Comment by iangregsondev** [comment]
It's not published in the maven repository as far as I can see. That's why I assumed it's probably being used from a local maven repo but never pushed. The main header client app has a dependency on CORE and It's being compiled somehow :-) Although it's not there.

But I do notice that there is a reference to the local maven repo in the example that was given i.e.

```
repositories {
    mavenCentral()
    mavenLocal()
}
```

I suppose I could compile the main core project and produce my own artifacts, but it's not something I wanted to dive into, to be truthful. 

Happy to test any fix or update - no worries, but right now, version 2 is not there.

**Comment by Fletch153** [comment]
@iangregsondev I've just published the HSV version 2.0.2 to Sonatype. The changes should be reflected on maven central within the next few hours.

**Comment by iangregsondev** [comment]
Thanks @Fletch153 , Oh I threw away my code :-(  Although I am eager to test it!

I will try and test it this weekend.

Thanks for getting back to me!

Looking forward to using it.
